cli/local/model_cache.go

```diff
@@ -134,6 +135,42 @@ func CacheModel(modelPath string, awsClient *aws.Client) (*spec.LocalModelCache,
 	return &localModelCache, nil
 }
 
+func DeleteCachedModels(apiName string, modelsToDelete []string) error {
+	errList := []error{}
+	modelsInUse := strset.New()
+	apiSpecList, err := ListAPISpecs()
+	errList = append(errList, err)
+
+	for _, apiSpec := range apiSpecList {
```

Comment:
This for loop should be wrapped in a:

```
if err !=nil {
   ...
}
```

The way it is written right now works well. I think for readability and not having issues, we should make certain that this code path of iterating through API specs is only run when there isn't an error.

Reply:
Done!